app/poros/quote.rb

```diff
@@ -0,0 +1,11 @@
+class Quote
+  attr_reader :content,
+              :author,
+              :tags
+
+  def initialize(quote_data)
+    @content = quote_data[:content]
+    @author = quote_data[:author]
+    @tags = quote_data[:tags]
+  end
+end
```

Comment:
👍 You could even add the id in here too which could help make tests more robust.

Reply:
^^ to expand a bit more: If you had an id associated with each quote, you could wrap each quote in an html id. In your test, you'd be able to iterate through the quotes, and make sure that within that quotes html id, that quotes information was present.